demo/dismissable/dismissable-item.js

```diff
@@ -44,16 +50,21 @@ class DismissableItem extends HTMLElement {
 
     this.wrapper = this.shadowRoot.querySelector('#contentWrapper');
 
+    this.scroller = null;
     this.position = 0;
     this.itemIndex = 0;
     this.width = 0;
     this.state = 'initial';
-    this.addEventListener('touchstart', this);
-    this.addEventListener('touchmove', this);
-    this.addEventListener('touchend', this);
-    this.addEventListener('pointerdown', this);
-    this.addEventListener('pointermove', this);
-    this.addEventListener('pointerup', this);
+    this.addEventListener('touchstart', this, {passive: true});
+    this.addEventListener('touchmove', this, {passive: true});
+    this.addEventListener('touchend', this, {passive: true});
+    this.addEventListener('pointerdown', this, {passive: true});
+    this.addEventListener('pointermove', this, {passive: true});
+    this.addEventListener('pointerup', this, {passive: true});
+  }
+
+  connectedCallback() {
+    this.scroller = this.closest("virtual-scroller");
```

Comment:
I think `this.scroller = this.offsetParent` would be better here. If null, we don't have any parent that sets a stacking context